[2022-06-25 19:42:04,213] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_26 scheduled__2022-06-24T19:41:59.779564+00:00 [queued]>
[2022-06-25 19:42:04,233] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_26 scheduled__2022-06-24T19:41:59.779564+00:00 [queued]>
[2022-06-25 19:42:04,233] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-06-25 19:42:04,233] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-06-25 19:42:04,233] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-06-25 19:42:04,255] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): START_COUNT_BY_DAY_26> on 2022-06-24 19:41:59.779564+00:00
[2022-06-25 19:42:04,262] {standard_task_runner.py:52} INFO - Started process 381 to run task
[2022-06-25 19:42:04,264] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'DM-COUNT-account-turnover', 'START_COUNT_BY_DAY_26', 'scheduled__2022-06-24T19:41:59.779564+00:00', '--job-id', '1349', '--raw', '--subdir', 'DAGS_FOLDER/dm_fill_account_turnover.py', '--cfg-path', '/tmp/tmp234lct5k', '--error-file', '/tmp/tmpyxx_wtod']
[2022-06-25 19:42:04,266] {standard_task_runner.py:77} INFO - Job 1349: Subtask START_COUNT_BY_DAY_26
[2022-06-25 19:42:04,459] {logging_mixin.py:109} INFO - Running <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_26 scheduled__2022-06-24T19:41:59.779564+00:00 [running]> on host 1a36ba6d0ffa
[2022-06-25 19:42:04,688] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=DM-COUNT-account-turnover
AIRFLOW_CTX_TASK_ID=START_COUNT_BY_DAY_26
AIRFLOW_CTX_EXECUTION_DATE=2022-06-24T19:41:59.779564+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-24T19:41:59.779564+00:00
[2022-06-25 19:42:04,713] {base.py:79} INFO - Using connection to: id: postgres. Host: postgres, Port: 5432, Schema: postgres, Login: ***, Password: ***, extra: {}
[2022-06-25 19:42:04,729] {dbapi.py:225} INFO - Running statement: CALL ds.fill_account_turnover_f (%(date)s::date);, parameters: {'date': '2018-01-26'}
[2022-06-25 19:42:05,215] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=DM-COUNT-account-turnover, task_id=START_COUNT_BY_DAY_26, execution_date=20220624T194159, start_date=20220625T194204, end_date=20220625T194205
[2022-06-25 19:42:05,264] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-25 19:42:05,749] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
